addons/binding/org.openhab.binding.deconz/ESH-INF/thing/thing-types.xml

```diff
@@ -61,11 +61,37 @@
 	</thing-type>
 
 	<channel-type id="power">
-		<item-type>Number:Energy</item-type>
+		<item-type>Number:Power</item-type>
 		<label>Power</label>
 		<description>Current power usage in Watts</description>
 		<state readOnly="true" pattern="%.1f %unit%"></state>
 	</channel-type>
+	
+	<thing-type id="consumptionsensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="deconz" />
+		</supported-bridge-type-refs>
+		<label>Consumption Sensor</label>
+		<description>A consumption sensor</description>
+		<channels>
+			<channel typeId="consumption" id="consumption"></channel>
+			<channel typeId="last_updated" id="last_updated"></channel>
+		</channels>
+		<config-description>
+			<parameter name="id" type="text" required="true">
+				<label>Sensor ID</label>
+				<description>The deCONZ bridge assigns an integer number ID to each sensor.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="consumption">
+		<item-type>Number:Energy</item-type>
+		<label>Consumption</label>
+		<description>Current consumption in Watts/hour</description>
```

Comment:
I am pretty sure the unit is correct.

From my point of view it does not make sense to add them to the description because with UoM they depend on the users default metric system.

Reply:
Well, i don't mind having units in there so every can see exactly what they will get when using the %unit% in the items definition, but i am also ok with not having units in there, it should just be consistent throughout the binding. In some decriptions are units, for example Lux for Brightness i think.

Reply:
don't know who should be deciding this ... i guess i could just implement the final decision and either put all units in there or none at all.

Reply:
Please remove them everywhere we use a QuantityType. Thanks.

Reply:
ok, all removed